app/views/govuk_publishing_components/components/_layout_for_public.html.erb

```diff
@@ -48,13 +50,24 @@
     <% unless omit_header %>
       <%= render "govuk_publishing_components/components/layout_header", {
         search: show_search,
-        # layout-header will always have border-bottom, unless the layout is full width
-        remove_bottom_border: full_width,
+
+        # The (blue) bottom border needs to be underneath the emergency banner -
+        # so it has been turned off and added in manually.
+        remove_bottom_border: true,
       } %>
     <% end %>
-    <div class="<%= "govuk-width-container" unless full_width %>"  id="content">
+
+    <%= raw(emergency_banner) %>
+
+    <% unless full_width %>
+      <div class="gem-c-layout-for-public__blue-bar govuk-width-container"></div>
+    <% end %>
+
+    <%= raw(global_bar) %>
+
+    <div id="wrapper" class="<%= "govuk-width-container" unless full_width %>">
```

Comment:
It's unfortunate that we'll have to carry this id over, but it is used all over our tests, including e2e.

Reply:
It took me an embarrassingly long while to realise that this was the reason that all the tests were failing!